### DISCUSSION TITLE
MQTT-messages on every run?

### DISCUSSION THREAD

**Discussion by binderth** [tillsteinbach/WeConnect-mqtt#314] [Q&A]
As I understand, the messages are only sent, if there's changes. The messages are "retained" for as long as they contain the same values from my experience...?

So my problem is: I'm using [evcc](https://github.com/evcc-io/evcc) for load-management and if I connect my ID.3 and evcc doesn't use the retained evcc-messages for vehicle information. Can I somehow configure WeConnect-mqtt to send messages on every run, regardless, if there are changes?

**Comment by tillsteinbach** [comment]
Yes, the option you are looking for is: `--republish-on-update`, but this will cause more traffic on your network of course
